class-tgm-plugin-activation.php

```diff
@@ -1179,6 +1127,65 @@ public function notices() {
 		}
 
 		/**
+		 * Generate the user action links for the admin notice.
+		 *
+		 * @since 2.x.x
+		 *
+		 * @param int $install_link_count  Number of plugins to install.
```

Comment:
Could these arg names be improved? If it's the number of plugins to install, why is it making the presumption that it's going to be used in a link? Name the variable for what it is, i.e. `$installable_count` or `$installable_plugin_count`.